src/ydata_profiling/model/alerts.py

```diff
@@ -77,16 +95,11 @@ class Alert:
     def __init__(
         self,
         alert_type: AlertType,
-        values: Optional[Dict] = None,
+        values: Optional[Dict] = {},
```

Comment:
This is dangerous because of the structure being mutable (https://www.inspiredpython.com/article/watch-out-for-mutable-defaults-in-function-arguments). It would be better to leave it as it was before.